src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/VectorConfig.java

```diff
@@ -15,27 +16,56 @@
  */
 public record VectorConfig(
     boolean vectorEnabled,
+    String fieldName,
     int vectorSize,
     SimilarityFunction similarityFunction,
     VectorizeConfig vectorizeConfig) {
 
   // TODO: this is an immutable record, this can be singleton
   // TODO: Remove the use of NULL for the objects like vectorizeConfig
   public static VectorConfig notEnabledVectorConfig() {
-    return new VectorConfig(false, -1, null, null);
+    return new VectorConfig(false, null, -1, null, null);
```

Comment:
What are the semantics of `null` vectorField? I assume we need a placeholder of some kind... ?
Put another way, how are we using this placeholder (how is it checked and so on).

Reply:
Gave null if the there is no vector field set for the table or collection